0000-Hex-Coverage-Limit.md

```diff
@@ -0,0 +1,101 @@
+
+## Motivation:
+HIP 74 was passed to incorporate obstruction data and radio signal power into the PoC reward model; however, it weighed all coverage within each res12 hex equally, even if multiple radios were already providing coverage within that res12 hex. This means deployers could point 5 outdoor radios in the same direction, and still be awarded full Modeled Coverage points for each res12 hex. 
+
+This proposal aims to improve the value of Mobile network coverage by incentivizing users to deploy radios that minimize overlapping coverage, and encourage deployments in new areas. 
+
+## Stakeholders:
+All Mobile Radio deployers/ Mobile Hotspot Owners. 
+ 
+## Detailed Explanation:
+Currently, any redundant and overlapping network coverage is still rewarded the same as non-overlapping coverage. This discourages the buildout of coverage to new areas. To prevent overcrowding and overlapping of coverage in hexes, this HIP proposes to limit the amount of modeled coverage points radios receive for redundant coverage in res12 hexes. 
+
+To ensure that only the best setups are rewarded, only the top four (4) radio signals in each res12 hex will be awarded modeled coverage points, with a decaying multiplier based on the radio score noted below. Any radios not scored within the top four (4) will be graded as “Fail”, and receive no modeled coverage points for that res12 hex. 
+
+
+| Score/Grade  |Multiplier|  
+|--------------|----------|
+|      1       |   1X     |
+|      2       |  .75X    |
+|      3       |  .5X     |
+|      4       |  .1X     |
+|    Fail      |   0X     |
+
+
+All outdoor radios that provide coverage to any res12 hex will be given a score for each res12 hex they provide coverage in based on the following potential attributes (note, this score is only for a single res12 hex and not the entire radio):
+
+•Mapped Signal Strength (Once mapping is implemented)*
+
+•Modeled Signal Strength 
+
+•Date of CPI approval (only used as a tiebreaker if tie for attribute 1)
+
+*Until Mapping is implemented, the only attributes used will be Modeled Signal Strength and date of CPI approval. See the end of this section for how Mapping is to be implemented for this HIP.
+
+
+Scoring attribute 2 (CPI approval date) will only be used when there is a tie for two or more radios for the proceeding attribute. 
+
+Please note, that the multiplier table above only affects the modeled coverage points that are given to each radio, and does not affect rewards distributed for the transfer of data. 
+
+See the example below of how this HIP would affect a deployment of five (5) radios that provide modeled coverage to the same res12 hex:
+
+
+| Radio |Signal Strength| CPI Approval Date | Score | Coverage Points Per HIP 74| New Coverage Points|  
```

Comment:
```suggestion
| Radio |Signal Strength| CPI Approval Date | Rank | Coverage Points Per HIP 74| New Coverage Points|  
```

calling this a score feels a little strange given its not used in the equation directly, maybe calling it "rank" makes a little bit more understandable?

Reply:
Changed to rank.